server.go

```diff
@@ -11,6 +11,17 @@ import (
 	"time"
 )
 
+var debug = os.Getenv("DEBUG")
```

Comment:
I'm not sure this is very idiomatic Go (i.e. using globals) but the environment variable is not going to change during runtime and the global saves querying the environment on every logDebug call.